### DISCUSSION TITLE
Does plan affect performance?

### DISCUSSION THREAD

**Discussion by DimaSergeew** [plan-player-analytics/Plan#2616] [APIs and Development]
I have questions to the developer of this particular plugin:
1) Does the plan affect the player's connection speed to the server?? does it slow down server performance?
2) If the hosting where Plan is installed does not have protection for web services and cannot protect the port that Plan uses, can it be connected to a remote web host?

**Comment by AuroraLS3** [comment]
1. No, Plan uses the mechanisms built-in the server APIs to gather data on a monitor level. Server performance should be unaffected.
2. HTTPs can be set up using self-signed certificates wherever it is is running even if you don't have a domain. In addition reverse-proxying is possible 
   - https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up
   - https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up#using-a-self-signed-certificate
   - https://github.com/plan-player-analytics/Plan/wiki/External-Webserver-use:-Reverse-proxy
